### DISCUSSION TITLE
How configure onedrive

### DISCUSSION THREAD

**Discussion by tkcaccia** [abraunegg/onedrive#2018] [General]
I have OneDrive 365 account 
I would like to configure onedrive to set "Business Shared Folders configured" parameter to true.
OneDrive does not synchronise folders shared by others.

Although, the function "onedrive --list-shared-folders" listed all shared folder with me, the folders created by Team groups are not listed.

**Comment by abraunegg** [comment]
@tkcaccia 
> I would like to configure onedrive to set "Business Shared Folders configured" parameter to true.
OneDrive does not synchronise folders shared by others.

To configure the client to sync OneDrive Shared Folders for Office365 | Business Accounts you must correctly configure the client. Simply setting `sync_business_shared_folders = "true"` in your configuration only enables the feature, it does not configure it. Please read https://github.com/abraunegg/onedrive/blob/master/docs/BusinessSharedFolders.md on how to configure this feature correctly.

> Although, the function "onedrive --list-shared-folders" listed all shared folder with me, the folders created by Team groups are not listed.

If these are Microsoft Teams Shared Folders, the OneDrive API does not handle them. You will need to use Microsoft Teams to access these.

If these are SharePoint Folders, you will need to reconfigure your client, or create a second client instance following https://github.com/abraunegg/onedrive/blob/master/docs/SharePoint-Shared-Libraries.md

If this is something else - please provide screenshots of of the 'Team groups' you are trying to access via the client.
